### PR TITLE
feat: style bignumber config

### PR DESCRIPTION
**Pull request by rephus**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #1983

### Description:
Add style to bignumber config 

### Preview:

Peek 2022-05-05 11-47

With format + rounding 
Peek 2022-05-05 10-37

### Scope of the changes (select all that apply):
- [x] Frontend  
- [x] Backend  
- [x] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

**Comment by rephus** [comment]
I refactored the label specific config from bignumber on VisualizationProvider and copied how we do it on carstesianConfig

File: packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ type VisualizationContext = {
     chartRef: RefObject<EChartsReact>;
     chartType: ChartType;
     cartesianConfig: ReturnType<typeof useCartesianChartConfig>;
+    bigNumberConfig: ReturnType<typeof useBigNumberConfig>;
```
Comment (ZeRego) [inline packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx:23]: We can remove the individual props (`bigNumber` `bigNumberLabel` etc,) from the context with this change 🙏

File: packages/frontend/src/hooks/useBigNumberConfig.ts [hunk 2]
```diff
@@ -34,29 +35,46 @@ const useBigNumberConfig = (
...
                 ? {
                       label: bigNumberLabel,
+                      style: bigNumberStyle,
                   }
                 : undefined,
-        [bigNumberLabel],
+        [bigNumberLabel, bigNumberStyle],
     );
-
+    console.log('validBigNumberConfig', validBigNumberConfig);
```
Comment (ZeRego) [inline packages/frontend/src/hooks/useBigNumberConfig.ts:69]: forgotten console log

File: packages/frontend/src/components/SimpleStatistic/index.tsx [hunk 3]
```diff
@@ -10,19 +11,29 @@ import {
...
     const validData = bigNumber && resultsData?.rows.length && bigNumberLabel;
 
     if (isLoading) return <LoadingChart />;
 
+    const formattedBigNumber = formatValue(
+        field?.format,
+        field?.round,
+        bigNumber,
+        bigNumberStyle,
+    );
```
Comment (ZeRego) [inline packages/frontend/src/components/SimpleStatistic/index.tsx:29]: I expected this to be done inside useBigNumberConfig so all the business logic is closer.

**Review by ZeRego — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/SimpleStatistic/index.tsx (hunk 3 above)
Comment (rephus) [reply packages/frontend/src/components/SimpleStatistic/index.tsx:29]: I suggested that on another thread, but I thought you would say that there should not be visualization logic inside the `config` or similar :) I don't mind moving the logic there if you think that is the right place.

File: packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx (hunk 1 above)
Comment (rephus) [reply packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx:23]: That's what I did already, right ? Did I miss anything ? https://github.com/lightdash/lightdash/pull/1999/files#diff-2714a1ae3d534b36cee59df77fa420e9008844c61dd318d29e6bcf5d95142166L27 and https://github.com/lightdash/lightdash/pull/1999/files#diff-2714a1ae3d534b36cee59df77fa420e9008844c61dd318d29e6bcf5d95142166L151

**Review by ZeRego — APPROVED** [review]
lgtm 👍 gj :)

**Pull request merged**